internal/metrics/cmd.go

```diff
@@ -154,6 +156,46 @@ func (m Metrics) getArch() string {
 	return strings.TrimSpace(string(b))
 }
 
+func (m Metrics) getHwCap() string {
+	// Make sure we have glibc version > 2.33
+	r := runCmd(m.libc6Cmd)
+	result, err := filterFirst(r, `^(?: +Installed: (.*))`, false)
+	if err != nil {
+		log.Infof("Couldn't get glibc version: "+utils.ErrFormat, err)
+		return ""
+	}
+	// This successfully handles a version of (none)
+	if strings.Compare(result, "2.33") < 0 {
+		return "N/A"
```

Comment:
Please, return rather "", that way, it’s removed from the json syntax, and we keep consistency with other fields and older releases (which would not have the field appearing)

Reply:
Thanks for the comments! In the original LP bug (https://bugs.launchpad.net/ubuntu/+source/ubuntu-report/+bug/1922712) it specifies that "The string should be "N/A" (not available), when trying to run ld-linux on a release with glibc (<< 2.33)"

I will speak with the bug reporter to make sure that is what we want before addressing the rest of these comments.